commons/src/test/java/com/powsybl/commons/report/ReportNodeTest.java

```diff
@@ -134,30 +134,34 @@ void testInclude() throws IOException {
                 .withMessageTemplate("rootTemplate", "Root message")
                 .build();
 
-        PowsyblException e1 = assertThrows(PowsyblException.class, () -> root.include(root));
-        assertEquals("Cannot add a reportNode in itself", e1.getMessage());
-
         ReportNode otherRoot = ReportNode.newRootReportNode()
                 .withMessageTemplate("includedRoot", "Included root message")
                 .build();
-        otherRoot.newReportNode()
+        ReportNode otherRootChild = otherRoot.newReportNode()
                 .withMessageTemplate("includedChild", "Included child message")
                 .add();
 
+        PowsyblException e1 = assertThrows(PowsyblException.class, () -> root.include(ReportNode.NO_OP));
+        PowsyblException e2 = assertThrows(PowsyblException.class, () -> otherRootChild.include(otherRoot));
+        PowsyblException e3 = assertThrows(PowsyblException.class, () -> otherRootChild.include(otherRoot));
```

Comment:
What is the purpose of testing it twice in a row?
If this test is really needed, can you add a comment explaining why?

Reply:
My mistake! the two lines were different before eb90bdf commit. Fixed